src/language-css/printer-postcss.js

```diff
@@ -458,16 +458,23 @@ function genericPrint(path, options, print) {
       for (let i = 0; i < node.groups.length; ++i) {
         parts.push(printed[i]);
 
-        // Ignore value inside `url()`
-        if (insideURLFunction) {
-          continue;
-        }
-
         const iPrevNode = node.groups[i - 1];
         const iNode = node.groups[i];
         const iNextNode = node.groups[i + 1];
         const iNextNextNode = node.groups[i + 2];
 
+        if (insideURLFunction) {
+          if (
+            (iNextNode &&
+              iNextNode.type === "value-operator" &&
+              iNextNode.value === "+") ||
```

Comment:
In theory we can have more operators (for example `-`, `*` and `/`), i think we should handle them too

Reply:
I think the additional operator is special because it is used for concatenation of string inside URL function.
So, we shouldn't handle `-`, `*`, `/`.